blog/settings.py

```diff
@@ -104,9 +109,22 @@
 
 USE_TZ = True
 
-
+LOGIN_REDIRECT_URL = '/'
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/1.10/howto/static-files/
 
 STATIC_URL = '/static/'
 STATIC_ROOT = os.path.join(BASE_DIR, 'static')
+
+DATABASES['default'] = dj_database_url.config()
+
+SECURE_PROXY_SSL_HEADER = ('HTTP_X_FORWARDED_PROTO', 'https')
+
+ALLOWED_HOSTS = ['*']
+
+DEBUG = False
+
+try:
+    from .local_settings import BASE_DIR, DATABASES, DEBUG  # noqa: F401
```

Comment:
There is one problem without `noqa`:
`blog/settings.py:128:5: F401 '.local_settings.DEBUG' imported but unused`